Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -155,6 +156,23 @@ def build_iterator(self, **kwargs):
 
         return results
 
+    def get_feed_content_divided_to_lines(self, url, raw_response):
+        """Fetch feed data and divides its content to lines
+
+        Args:
+            url: Current feed's url.
+            raw_response: The raw response from the feed's url.
+
+        Returns:
+            List. List of lines from the feed content.
+        """
+        if self.feed_url_to_config and self.feed_url_to_config.get(url).get('is_zipped_file'):  # type: ignore
+            response_content = gzip.decompress(raw_response.content)
```

Comment:
I think by using the `.decompress()` method, you could still run into the issue I opened [here](https://github.com/demisto/etc/issues/22325) of a memory error. Test against the dga zip and see - but if it does give you memory errors as I suspect, then look at using `gzip.open`.

Reply:
Ive tested all of them locally and it seemed to work just fine.
Do you think it still would be better doing it in another way?

Reply:
See this [comment](https://github.com/demisto/etc/issues/22325#issuecomment-602117258) - if everything is working as is and you don't want to implement the enhancement right now, I'll still approve the PR after the updates to the mapping.

Reply:
I read about it for a little, but it doesnt seem to be different since I have to receive the full request and then read it fully.
So, for now it seems to be working fine so Ill keep it that way.